spec/ttml1.xml

```diff
@@ -4324,26 +4322,57 @@ glyphs do not have an EM square that is distinct from their font's EM square.</p
 <p>If horizontal and vertical sizes are expressed independently, then the
 units of the <loc href="#style-value-length">&lt;length&gt;</loc> values must be the same.</p>
 <p>The <loc href="#style-value-length">&lt;length&gt;</loc> value(s) used to express font size must be non-negative.</p>
+
+<p>Relative <loc href="#style-value-length">&lt;length&gt;</loc> values
+that appear in this attribute, i.e., values expressed in percentage (<code>%</code>), cell (<code>c</code>), or EM (<code>em</code>) units,
+are resolved in relation to a referenced 2-dimensional size value consisting of two components, a width component and
+a height component.</p>
```

Comment:
Missing the following text present in TTML2:

>If the relative unit is a percentage, then the referenced value is either the computed value of the font size of the nearest styled ancestor element or the computed cell size. If the relative unit is cells (c), then the referenced value is the computed cell size. If the relative unit is EMs (em), then the referenced value is determined as if percentage units were used, where 1em equals 100%.

Reply:
First sentence is already in `tts:fontSize` definition table, and the two following sentences are in fact in the prose.

Reply:
I would prefer that the order of the two notes "Anamorphic ..." and "A glyph's EM square ..." be changed so the latter appears first. This corresponds with the material in the preceding paragraph: EM square is used before referring to anamorphic (two values).